http/headers/Accept-Ranges.json

```diff
@@ -7,14 +7,14 @@
           "spec_url": "https://www.rfc-editor.org/rfc/rfc9110#field.accept-ranges",
           "support": {
             "chrome": {
-              "version_added": true
+              "version_added": "≤56"
```

Comment:
This is a header sent by the server, and it's used to resume downloads and to detect if it's possible to seek in `<video>`. I think we can set it to "1" everywhere based on the assumption that resuming downloads has been possible in all browsers since Safari 1.

Reply:
https://github.com/mdn/browser-compat-data/pull/23720